javascript/javascript/examples/ImportAndExport.js

```diff
@@ -0,0 +1,8 @@
+// https://github.com/antlr/grammars-v4/issues/3484
```

Comment:
We could use MDN examples here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#syntax
https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#syntax
Also I would separate them into two files `import.js` and `export.js`.

Reply:
Yes, _we_ coulld! 😉